packages/frontend/src/components/FileInput/styles.css

```diff
@@ -0,0 +1,9 @@
+.file-input {
```

Comment:
Khanh knows more about this stuff, but why is this as a css file while some other stuff is inline and some is using styled? Not sure if this is on purpose or not, just something that caught my eye.

Reply:
Solved this by using MUI TextField component